### DISCUSSION TITLE
image example

### DISCUSSION THREAD

**Discussion by elrumordelaluz** [pmndrs/leva#322] [Q&A] [answered]
First of all thanks for this useful and beautiful GUI!

I am wondering if there is a simple example on how to implement the `image` input like [this one](https://leva.pmnd.rs/?path=/story/inputs-image--image).

thanks!

**Comment by dbismut** [comment, marked as answer]
Not sure about what you need…

**Reply by elrumordelaluz** [reply]
didn't find a way to implement this _image_ input control:

<img width="320" alt="Screenshot 2022-03-24 at 13 26 48" src="https://user-images.githubusercontent.com/784056/159916076-915cff77-e1c7-40ab-a558-4e97ff491bd7.png">

**Reply by gsimone** [reply]
Just do 

```
const { foo } = useControls({ foo: { image: undefined } })
```

**Reply by elrumordelaluz** [reply]
Really don't know why when tried this before was getting me a _plugin_ error, but now works. Thank you!

**Reply by dbismut** [reply]
Haha ok, as this is the code inside the storybook I wasn't sure about the question ;)
